.github/workflows/build-ros2-ws-action.yml

```
@@ -0,0 +1,79 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [master]  
+jobs:
+  build-and-test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4.1.1
+        with:
+          path: aruku
+          ref: feature/fix-using-grpc
+
+      - name: Checkout Kansei
+        uses: actions/checkout@v4.1.1
+        with:
+          repository: ichiro-its/kansei          
+          path: kansei
```

Quick question, is it necessary to set the `path` if the value is the same as the repository name? Can you try this instead and see if it behaves the same?
```suggestion
      - name: Checkout Kansei
        uses: actions/checkout@v4.1.1
        with:
          repository: ichiro-its/kansei
```